src/lib.rs

```diff
@@ -431,6 +452,17 @@ impl<S: SuffixScheme> FileRotate<S> {
                                 self.count = 0;
                             }
                         }
+                        ContentLimit::Time(_) => {
+                            if let Ok(metadata) = file.metadata() {
+                                self.modified = match metadata.modified() {
```

Comment:
What happens on systems where `nomtime` (no modification time) is specified?

Reply:
Python is using mtime to, for log rotation, and it runs also on multiple platform. So I guess, the most common system will work. But maybe we found a fall-back?